packages/web-react/src/types/grid.ts

```diff
@@ -0,0 +1,22 @@
+import { ElementType, JSXElementConstructor } from 'react';
+import { ChildrenProps, StyleProps } from './shared';
+
+export type GridColumns = 1 | 2 | 3 | 4 | 6 | 12;
+
+export interface GridElementTypeProps<T extends ElementType = 'div'> {
+  /**
+   * The HTML element or React element used to render the Grid, e.g. 'div'.
+   * @default 'div'
+   */
+  elementType?: T | JSXElementConstructor<any>;
+}
+
+export interface GridProps<T extends ElementType = 'div'> extends GridElementTypeProps<T> {}
+
+export interface SpiritGridProps<T extends ElementType = 'div'> extends GridProps<T>, ChildrenProps, StyleProps {
+  mobile?: GridColumns;
```

Comment:
This one can be left out. There is no such CSS class as `Grid--mobile--cols-*` because there is already `Grid--cols-*` which does the (mobile-first) job. The `cols` prop can be listed first then because it will be the go-to option in most cases (supposed the order here is not alphabetical).

Reply:
Well, I did not notice, because I saw `$breakpoints` in tokens and generate all variants. Thanks for pointing on it.